sdk/src/main/java/io/customer/sdk/data/store/ApplicationStore.kt

```diff
@@ -16,6 +19,8 @@ internal class ApplicationStoreImp(val context: Context) : ApplicationStore {
         get() = appInfo.first
     override val customerAppVersion: String
         get() = appInfo.second
+    override val isPushSubscribed: Boolean
+        get() = NotificationManagerCompat.from(context).areNotificationsEnabled()
```

Comment:
This seems like a good idea to me. However, it doesn't match the plan that the squad thought up in the notion doc. So, I approve of this implementation however, I would send a message to the squad saying that this is how `push_subscribed` device attribute is going to work + update the notion doc table so LJ knows about this implementation.

Reply:
> In my last PR review, I left this comment
> 
> > In this PR, I don't see support for adding custom attributes? Is that planned for a future PR? On iOS, it got implemented in it's PR for adding device attributes. I assume Android should support it as well?
> 
> I don't see support for adding custom attributes added so this question still stands. Besides this question, the PR looks good to me.

Awesome thanks @levibostian ! `custome attributes` is part of future releases when BE/FE supports it.